packages/cli/src/utils/api.js

```diff
@@ -84,7 +82,9 @@ const callAPI = (route, options, rawError = false) => {
         }
       }
 
-      if (constants.DEBUG || global.argOpts.debug) {
+      // TODO: pull this into the command? to easily access debug opt
+      // or we do better at using regular node debug lib
+      if (DEBUG || global.argOpts.debug) {
```

Comment:
We should probably use [debug](https://github.com/visionmedia/debug), which will let us not have to pass global args around (which is another area I wanted to address)